ducktape/cluster/remoteaccount.py

```diff
@@ -28,11 +28,15 @@ def __init__(self, hostname, user=None, ssh_args=None, ssh_hostname=None, logger
         self.ssh_args = ssh_args
         self.ssh_hostname = ssh_hostname
         self.externally_routable_ip = None
-
         self.logger = logger
 
-    def set_logger(self, logger):
-        self.logger = logger
+    def __repr__(self):
+        r = ["hostname: %s" % self.hostname,
+             "user: %s" % ("" if self.user is None else self.user),
+             "ssh_hostname: %s" % ("" if self.ssh_hostname is None else self.ssh_hostname),
```

Comment:
I assume you're using `__repr__` because containers use it for elements in their `__str__`? Not sure we want to use the default for containers here. This is going to get pretty noisy if you include all of this (especially ssh_args, which has a *lot* of flags under Vagrant). Wouldn't just printing out a str representation with a couple of pieces of info be enough?